server-timing/idlharness.any.js

```diff
@@ -0,0 +1,22 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+// https://w3c.github.io/server-timing/
+
+promise_test(async () => {
+  const idl = await fetch('/interfaces/server-timing.idl').then(r => r.text());
+  const res = await fetch('/interfaces/resource-timing.idl').then(r => r.text());
+  const perf = await fetch('/interfaces/performance-timeline.idl').then(r => r.text());
+
+  const idl_array = new IdlArray();
+  idl_array.add_idls(idl);
+  idl_array.add_dependency_idls(res);
+  idl_array.add_dependency_idls(perf);
+  idl_array.add_objects({
+    Performance: ['performance'],
```

Comment:
@npm1 - how would I go about getting a `serverTiming` object for the IDL harness to test?
I was looking at https://developer.mozilla.org/en-US/docs/Web/API/PerformanceServerTiming and tried 

    performance.getEntriesByType('resource')[0].serverTiming

But, it's empty (presumably because the headers aren't attached to the response).

Reply:
Maybe try performance.getEntriesByType('navigation')[0].serverTiming

Reply:
Also doesn't exist. Might land this as-is and add it (an object) at a later date.